docs/src/contracts/the-setup-contract-test-macro.md

```diff
@@ -0,0 +1,18 @@
+# The setup_contract_test! macro
+
+When deploying contracts with the `abigen!` macro, as shown in the previous sections, the user has the ability to change the default configuration parameters, lunch several providers, create multiple wallets, create specific assets, etc. However, it is often the case that we want to test only the contract methods and we want to simply deploy the contract with the default configuration parameters. The `setup_contract_test!` macro does exactly that. When expanded, the `setup_contract_test!` macro will: run the abigen, launch a local provider, setup one wallet and deploy the selected contract. The setup code that you have seen in previous sections gets reduced to:
```

Comment:
```suggestion
When deploying contracts with the `abigen!` macro, as shown in the previous sections, the user can:
* change the default configuration parameters, 
* launch several providers, 
* create multiple wallets, 
* create specific assets, etc. 

However, it is often the case that we want to test only the contract methods and we want to simply deploy the contract with the default configuration parameters. The `setup_contract_test!` macro does exactly that. When expanded, the `setup_contract_test!` macro will: 
1. run the abigen, 
2. launch a local provider, 
3. setup one wallet 
4. deploy the selected contract. 
The setup code that you have seen in previous sections gets reduced to:
```

Reply:
Albeit you probably shouldn't commit the suggestion as I've typed it, cuz I've wrapped it to fit a ~100char column. Although the final output is the same, I recall somebody having an issue with the line breaks. Can't remember the details.

In any case the suggestion was `lunch` vs `launch` and the enumerated and unenumerated lists.